### PR TITLE
V3 package source for tests

### PR DESCRIPTION
**Pull request by agr**

### CODE REVIEW

File: tests/NuGet.Config [hunk 1]
```diff
@@ -8,10 +8,14 @@
     <add key="automatic" value="True" />
   </packageRestore>
   <packageSources>
-    <add key="v2.nuget.org" value="https://nuget.org/api/v2" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (joelverhagen) [inline tests/NuGet.Config:11]: We should have clear here too.

File: tests/NuGet.Config [hunk 2]
```diff
@@ -8,10 +8,14 @@
...
     <add key="nugetbuild" value="https://dotnet.myget.org/F/nuget-build/api/v2/" />
   </packageSources>
   <activePackageSource>
     <add key="All" value="(Aggregate source)" />
   </activePackageSource>
+  <disabledPackageSources>
+    <clear />
+    <add key="https://www.nuget.org/api/v2/" value="true" />
```
Comment (joelverhagen) [inline tests/NuGet.Config:19]: Why disable this?

File: tests/NuGet.Config (hunk 1 above)
Comment (agr) [reply tests/NuGet.Config:12]: Added

File: tests/NuGet.Config (hunk 2 above)
Comment (agr) [reply tests/NuGet.Config]: Copied from other NuGet.configs we have here and there.

File: tests/NuGet.Config [hunk 3]
```diff
@@ -8,10 +8,14 @@
     <add key="automatic" value="True" />
   </packageRestore>
   <packageSources>
-    <add key="v2.nuget.org" value="https://nuget.org/api/v2" />
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="nugetbuild" value="https://dotnet.myget.org/F/nuget-build/api/v2/" />
```
Comment (joelverhagen) [inline tests/NuGet.Config:13]: nit: v3 here too 🥺 ?

Comment (joelverhagen) [reply tests/NuGet.Config:13]: Not a big deal, just if you feel like it.

**Pull request merged**